packages/components/src/Combobox/components/ComboboxContent/ComboboxContentSearch/ComboboxContentSearch.css

```diff
@@ -8,8 +8,10 @@
   padding-right: calc(var(--space-large) * 2.25);
   border: none;
   border-bottom: var(--border-base) solid var(--color-border);
+  color: var(--field-value-color);
```

Comment:
because this input is not coming from FormField, it needs us to tell it how to look!